pkg/codegen/python/gen.go

```diff
@@ -1941,15 +1941,14 @@ func genPulumiPluginFile(pkg *schema.Package) ([]byte, error) {
 	plugin := &plugin.PulumiPluginJSON{
 		Resource: true,
 		Name:     pkg.Name,
-		Version:  "${PLUGIN_VERSION}",
```

Comment:
Nice. I like this. Now, when this codegen is adopted by the providers we aren't forced to make a change to the `Makefile` to search/replace this value. We can do it at our leisure. And we can use `jq` to add the `version` property.

I do think we'll end up wanting to update our provider `Makefiles` to do this, because it'll fix being able to automatically download plugins for dev builds of the package, but that's not a huge priority.